WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1271,13 +1271,16 @@ decisions = {
 		potential = {
 			has_dlc = "Sons of Abraham"
 			is_playable = yes
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			k_israel = {
 				has_holder = no
 			}
 		}
 		allow = {
-			completely_controls = k_oriens
+			completely_controls = d_iudea # K_ORIENS IS TOO BIG, THIS IS WHAT ISRAEL HAD IN RL
```

Comment:
@IhateTrains 

That would require a complete overhaul of the existing Regions. It's better just to use what we already have: "D_IUDEA" almost matches the EXACT borders of the real Kingdom of Israel.

I EVEN PROVIDED A MAP. Why won't Loup give in when I gave accurate sources proving Israel was that size?!